WordPress/gutenberg
# [RNMobile] Fix jumping toolbar

## Description

Fix jumping toolbar after closing the `LinkSettings` when Button/Paragraph is focused

Fixes: https://github.com/wordpress-mobile/gutenberg-mobile/issues/2301

## How has this been tested?

1. Add a Button
2. Click on Button block settings
3. Focus on URL field
4. Close the Bottom sheet by tapping outside of it
5. Expect: toolbar mustn't jump

Test as well [Buttons-1-2-3](https://github.com/wordpress-mobile/test-cases/blob/master/test-suites/gutenberg/sanity-tests.md)

## Screenshots 

iphone 8 | iphone 11
--- | ---
![jump_toolbar_ok](https://user-images.githubusercontent.com/22746080/86575125-0f7bb380-bf77-11ea-8fb9-dd196057b351.gif) | ![keyboard_iX_2](https://user-images.githubusercontent.com/22746080/86575050-f4a93f00-bf76-11ea-9520-1de57843d41e.gif)


## Types of changes





## Checklist:
